Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,886 @@
+        .INPUTS
+        None. You cannot pipe objects to Set-SQLServerRequiredTLSVersion.
+        .OUTPUTS
+        None. Set-SQLServerRequiredTLSVersion does not return anything that can be piped and used as an input to another command.
+        .EXAMPLE
+        PS> Set-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -DryRun
+        .EXAMPLE
+        PS> Set-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck
+        .EXAMPLE
+        PS> Set-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\setMinTLSVersionForSQLServers\SQLServersWithoutMinReqTLSVersion.csv
+        .LINK
+        None
+    #>
+
+    param (
+        [String]
+        [Parameter(ParameterSetName = "DryRun", Mandatory = $true, HelpMessage="Specifies the ID of the Subscription to be remediated")]
+        [Parameter(ParameterSetName = "WetRun", Mandatory = $true, HelpMessage="Specifies the ID of the Subscription to be remediated")]
+        $SubscriptionId,
+
+        [Switch]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies a forceful remediation without any prompts")]
+        $Force,
+
+        [Switch]
+        [Parameter(ParameterSetName = "DryRun", HelpMessage="Specifies validation of prerequisites for the command")]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies validation of prerequisites for the command")]
+        $PerformPreReqCheck,
+
+        [Switch]
+        [Parameter(ParameterSetName = "DryRun", Mandatory = $true, HelpMessage="Specifies a dry run of the actual remediation")]
+        $DryRun,
+
+        [Switch]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies no back up will be taken by the script before remediation")]
+        $SkipBackup,
+
+
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the path to the file to be used as input for the remediation")]
+        $FilePath,
+
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the path to the file to be used as input for the remediation when AutoRemediation switch is used")]
+        $Path,
+
+        [Switch]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies script is run as a subroutine of AutoRemediation Script")]
+        $AutoRemediation,
+
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the time of creation of file to be used for logging remediation details when AutoRemediation switch is used")]
+        $TimeStamp
+    )
+
+    Write-Host $([Constants]::DoubleDashLine)
+    Write-Host "[Step 1 of 4] Prepare to set required TLS version for SQL Servers in Subscription: [$($SubscriptionId)]"
```

Comment:
Kindly refer to this BRS - https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1 - and make changes accordingly in the following code block.

Reply:
Changed following above script